kube-fluentd-operator.yaml

```diff
@@ -16,6 +16,7 @@ package:
       - merged-bin
       - ruby-3.2
       - ruby3.2-bundler
+      - systemd
```

Comment:
this does not seem right.

systemd should not be pulled into the container or running in the container.

does the new dependency execute some binaries from that package? or does it require services to be running from it?